core/types/rlp_payload.libevm.go

```diff
@@ -176,6 +203,43 @@ func (ExtraPayloads[HPtr, BPtr, SA]) cloneStateAccount(s *StateAccountExtra) *St
 	}
 }
 
+// blockOrBody is an interface for use as a method argument as they can't
+// introduce new generic type parameters.
+type blockOrBody interface {
+	isBlockOrBody() // noop purely for tagging
```

Comment:
Can you clarify what this is for? 👀 🤔 

I suppose it is to restrict argument types passed to cloneBlockOrBodyPayload right? If so, it would be nice to comment it

```suggestion
	isBlockOrBody() // noop to restrict argument type in [cloneBlockOrBodyPayload]
```

Reply:
Making the change in my review-response PR to use slightly different wording.